docs/howto/operate/index.md

```diff
@@ -14,6 +14,7 @@ new-cluster/index.md
 k8s/index.md
 delete-hub.md
 manual-deploy.md
+ci-cd.md
```

Comment:
Given where you've put this, it feels like we could split this page into two parts:

1. Some kind of "set up an AWS hub for out CI/CD" guide - probably part of our "Deploy a new hub" docs.
2. A high-level explainer of our CI/CD in general, perhaps as a page in our reference documentation.

What do you think?